src/nat/cli/commands/red_teaming/red_teaming.py

```
@@ -0,0 +1,147 @@
+    help="Override config values for the base workflow config using dot notation "
+         "(e.g., --override llms.nim_llm.temperature 0.7)",
+)
+@click.pass_context
+def red_team_command(ctx, **kwargs) -> None:
+    """Run red teaming evaluation with multiple scenarios."""
+    pass
+
+
+@red_team_command.result_callback(replace=True)
+def process_red_team_eval(
+    processors,
```

Ruff (0.14.8)

Unused function argument: `processors`

(ARG001)